### ISSUE TITLE
Test for WCAG success criterion 3.2.4: Consistent Identification

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#193]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html
Components that have the same functionality within a set of Web pages are identified consistently. 

The intent of this Success Criterion is to ensure consistent identification of functional components that appear repeatedly within a set of Web pages. A strategy that people who use screen readers use when operating a Web site is to rely heavily on their familiarity with functions that may appear on different Web pages. If identical functions have different labels on different Web pages, the site will be considerably more difficult to use. It may also be confusing and increase the cognitive load for people with cognitive limitations. Therefore, consistent labeling will help.

This consistency extends to the text alternatives. If icons or other non-text items have the same functionality, then their text alternatives should be consistent as well.

If there are two components on a web page that both have the same functionality as a component on another page in a set of web pages, then all 3 must be consistent. Hence the two on the same page will be consistent.

While it is desirable and best practice always to be consistent within a single web page, 3.2.4 only addresses consistency within a set of web pages where something is repeated on more than one page in the set.
## Applicable

Always (when a website is more than one page)
## Scope selector

Set of pages with inside
- icons
- links
  ..
## Failures
- [ ] F31: Using two different labels for the same function on different Web pages within a set of Web pages ( #194 ) 
## Techniques
- [ ] G197: Using labels, names, and text alternatives consistently for content that has the same functionality ( #195) AND following the sufficient techniques for Success Criterion 1.1.1 and sufficient techniques for Success Criterion 4.1.2 for providing labels, names, and text alternatives.
## Relationships
- Does this success criterion only pass if 1.1.1 and 4.1.2 pass?
- F31 is the counterpart of G197, although F31 is talking about a set of pages and G197 of content within a page.
- we can measure repeated patterns on pages (G197), but is it harder to detect inconsequences (F31)? Probably we could use form-ids, and for icons, use their hash and see if they are used in different situations.
- we need to pass DOM's of other pages to make a comparison possible.
